style.css

```diff
@@ -3,16 +3,16 @@ body {
 }
 
 .container {
-  width: 381px;
+  width: 100%;
   height: 6301px;
   padding-bottom: 118px;
   margin-bottom: 59px;
 }
 
```

Comment:
- [OPTIONAL] It would be nice to make your mobile version of the portfolio responsive and not have horizontal scrolling. It can cause accessibility issues when users try to access your portfolio. 

![chrome-capture-2023-2-30](https://user-images.githubusercontent.com/77106962/228957969-fd1e6fd2-c74e-4416-9b45-d6adefce1d8e.gif)